services/app-web/src/pages/App/App.tsx

```diff
@@ -70,4 +73,6 @@ function App({
     )
 }
 
-export default App
+export default withLDProvider<AppProps>({
```

Comment:
my vote would be to move this call into index.tsx. That way we’d be checking for the config in there and immediately using it in wrapping the App component. It also decouples LD from the app so if we needed to host App in a test or something we still could eaily.

Reply:
Ok so moving it over into index.tsx seems to have broken things, I'll see what's going on with it.

Reply:
Ok I changed this over to using the async provider LD has -- seems to be working again from `index.html` via the async init.